src/JustSaying/Fluent/TopicPublicationBuilder`1.cs

```diff
@@ -111,6 +116,23 @@ public TopicPublicationBuilder<T> WithTag(string key, string value)
         return this;
     }
 
+
+    /// <summary>
+    /// Configures the name of the topic
+    /// </summary>
+    /// <param name="name">The name of the topic to publish to.</param>
+    /// <returns>
+    /// The current <see cref="TopicSubscriptionBuilder{T}"/>.
+    /// </returns>
+    /// <exception cref="ArgumentNullException">
+    /// <paramref name="name"/> is <see langword="null"/>.
+    /// </exception>
```

Comment:
This isn't being thrown. Should it or should the comment be deleted?

Reply:
Ah thanks, yep it should be thrown. Added 👍